src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How manage your alerts and notifications"
+---
+
+import AQMdash from 'images/alerts_screenshot-crop_AQM-dash.webp'
+
+import AQMflow from 'images/alerts_diagram_AQM-flow.webp'
+
+As time goes on, your number of alerts will grow. This can lead to problems for your organization if they're not managed correctly. Additionally, your alerts will give you crucial information you can use to improve your system, and if you don't take advantage of that information, you won't be using your alerts to their fullest potential. Below, we've provided information on managing the quality of your alerts to prevent things like alert fatigue, as well as how you can use alerts to gather data and drive positive impact to your organization.
+
+<img
+    title="AQM dashboard"
+    alt="A screenshot displaying a view of an AQM dashboard in New Relic"
+    src={AQMdash}
+/>
+
+## Manage alert quality
+
+Alert quality management (AQM) is a concept focused on reducing unneccessary alerts and helping you ensure the alerts you receive are the most relevant ones. AQM follows a simple flow of installing the dashboard, getting information to establish your baselines, gathering data over a period of time, and then making changes based on that data to improve your stack and drive positive impact for your organization.
+
+<img
+    title="AQM flow"
+    alt="A diagram displaying how to use AQM to improve your system"
+    src={AQMflow}
+/>
+
+To get started with AQM, you need to do the following:
+
+<CollapserGroup>
+    <Collapser
+        id="AQM-dash"
+        title="1. Install the AQM dashboard"
+    >
+1. [Go to the **Alert Quality Management** instant observability page](https://newrelic.com/instant-observability/alert-quality-management)
+2. Click on **Install now**
+3. Choose an account to install the dashboard into
+4. View your dashboard
+    </Collapser>
+    <Collapser
+        id="KPIs"
+        title="2. Analyze your KPIs"
+    >
+With the dashboard installed, you can analyze key performance indicators, also known as KPIs. These KPIs are:
+
+* Incident count (alerts with a high number of incidents)
+* Accumulated incident time (alerts with high cumulative durations)
+* Mean time to close (the amount of time it takes until incidents are closed)
+* Percent under 5 minutes (the amount of incidents open for less than 5 minutes)
+
+The **Alerting Count by Policy** pane in the dashboard helps you identify these alert policies and determine any relevant patterns.
+    </Collapser>
+    <Collapser
```

Comment:
I think you have two procedures here jammed into one:

* Steps 1 and 2 are *purely procedural* things that *one person can do*. They're about how to access and use a UI
* Steps 3 thru 5 are *conceptual* things that *require organizational action and buy-in*

Combining them is leading to some problems:

* It doesn't show that the reader can get a win by doing _just_ steps 1 and 2 and nothing else
* It creates a very abrupt transition between simple procedural stuff and complex content
* It denies you the chance for a clearer lead-in to the organizational business